src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+function doNumericStyle(num) {
+  const numStr = num.toString();
+  const arr = [];
+
+  for (let i = numStr.length; i > 0; i -= 3) {
+    arr.push(numStr.slice(Math.max(0, i - 3), i));
+  }
+
+  return arr.reverse().join(',');
+}
+
+const populationSpan = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const populations = Array.from(populationSpan).map((span) => {
+  return +span.textContent.replaceAll(',', '');
```

Comment:
The `replaceAll` method is not supported in all environments. Consider using `replace` with a regular expression: `span.textContent.replace(/,/g, '')`.